.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "plugins": ["@babel/plugin-proposal-class-properties"]
+}
```

Comment:
Oh, no newline at the and of file…

Maybe you should use editorconfig.org and plugin for editor with support.

Reply:
Sure, i'll create new PR with code styling, eslint after that PR will be done.